pkgs/top-level/all-packages.nix

```diff
@@ -20053,6 +20053,27 @@ with pkgs;
 
   kobodeluxe = callPackage ../games/kobodeluxe { };
 
+  lc0-cuda = callPackage ../games/lc0 {
```

Comment:
I think it makes sense to do a "vanilla" lc0 and then do these others as overrides.